README.md

```diff
@@ -74,6 +74,16 @@ the meantime, you can use
 [brianloveswords/base64url](https://github.com/brianloveswords/base64url)
 to decode the signature.
 
+As of nodejs `v0.11.8`, SPKAC support was introduce. If your node
+version satisfies, then you can pass an object 
+
+```javascript
```

Comment:
(nit) Mind just making this inline with the sentence? It takes up quite a bit of visual space without providing a whole ton of value